src/core_plugins/kbn_vislib_vis_types/public/controls/point_series_options.js

```diff
@@ -9,6 +9,31 @@ module.directive('pointSeriesOptions', function ($parse, $compile) {
   return {
     restrict: 'E',
     template: pointSeriesOptionsTemplate,
-    replace: true
+    replace: true,
+    link: function ($scope, $el) {
+      $scope.showSeperateYAxisOption = () => {
+        const result = (() => {
+          if ($scope.vis.params.setYExtents || $scope.vis.params.defaultYExtents) return false;
+          if ($scope.vis.type.name === 'histogram' && $scope.vis.params.mode !== 'grouped') return false;
+          if ($scope.vis.type.name === 'area') return false;
+
+          let metrics = 0;
+          let buckets = 0;
+          _.map($scope.vis.aggs, agg => {
+            if (agg.type && agg.type.type === 'metrics') metrics++;
+            else buckets++;
+          });
+
+          if (metrics < 2 || buckets !== 1) return false;
```

Comment:
only show this option if we have 1 bucket only (doesn't make much sense with multiple buckets ...)